x/tss/keeper/grpc_query.go

```diff
@@ -58,3 +62,71 @@ func (q Querier) AssignableKey(c context.Context, req *types.AssignableKeyReques
 
 	return &types.AssignableKeyResponse{Assignable: !assigned}, nil
 }
+
+// ValidatorKey returns a map of active multisig role key ids to pub keys of a validator
+func (q Querier) ValidatorKey(c context.Context, req *types.ValidatorKeyRequest) (*types.ValidatorKeyResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+
+	valAddress, err := sdk.ValAddressFromBech32(req.Address)
+	if err != nil {
+		return nil, err
+	}
+
+	validator := q.staker.Validator(ctx, valAddress)
```

Comment:
Why? unbonded validators are still allowed and encouraged to sign.

Reply:
This doesn't restrict it. An unbonded validator is still considered a `Validator` and has an associated keeper state. Reminded me of another rewards issue, we don't reward unbonded validators who're still holding active key shares.

Reply:
I still don't see why is this necessary to check.